style.css

```diff
@@ -146,6 +146,18 @@ h3 {
   margin-bottom: 12px;
 }
 
+.mobile-menu ul {
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  color: #fff;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 0 24px;
+}
```

Comment:
- [ ] You did a great job on the styles but you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A838&t=M4yJZwaDfMJ0LtMC-0) template as much as possible. Please fix the below issues to match the design with the Figma template.
  - Please make sure that the background is not transparent.
  - Please use the correct color for mobile menu options `#FFF5E1`.

### Your design

![image](https://user-images.githubusercontent.com/83803414/208882098-37c65291-b288-410a-ab01-ff2cb593777d.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/208882049-34dc1c74-f758-4fb4-bc31-f7054f098d75.png)